webui/react/src/pages/Cluster/ClusterOverallStats.tsx

```diff
@@ -18,25 +20,25 @@ import { Loadable } from 'utils/loadable';
 
 import { maxClusterSlotCapacity } from '../Clusters/ClustersOverview';
 
+const EXPERIMENTS_PARAMS: Readonly<GetExperimentsParams> = { limit: -2, states: activeRunStates };
```

Comment:
Tiny change, but can you set this to ACTIVE_ EXPERIMENTS_PARAMS ? I was looking for where the active experiments were accounted for in the file

Reply:
@mapmeld how come `limit: -2`?

Reply:
From the Swagger docs: `[limit] -2 - returns pagination info but no experiments.`

Reply:
thanks, Emily. i def need to add comment